devito/operator.py

```diff
@@ -198,7 +198,9 @@ def _dle_arguments(self, dim_sizes):
         dle_arguments = OrderedDict()
         autotune = True
         for i in self.dle_arguments:
-            dim_size = dim_sizes.get(i.original_dim.name, i.original_dim.size)
+            dim_size = dim_sizes.get(i.original_dim.name,
+                                     i.original_dim.size if i.original_dim.is_Fixed
```

Comment:
Perhaps we don't need this ``if``, because if ``False`` it's gonna return ``None`` anyway?

Reply:
@navjotk